src/stores/game.js

```diff
@@ -391,7 +391,7 @@ export const useGameStore = defineStore('game', {
         case 'seven/targetedOneOff':
         case 'pass':
           // add all the move-making ones here
-          return `/api/game/${this.gameId}/move/${slug}`;
+          return `/api/game/${this.id}/move/${slug}`;
```

Comment:
This was an old mistake I made, the game store doesn't have `gameId`, it uses `id`

Reply:
Was it working before? If not it feels like we have a testing gap here.

Reply:
It was not, we're not far enough in the gameState api to actually hit any of this.